contrib/slime-repl.el

```diff
@@ -280,13 +280,14 @@ This is set to nil after displaying the buffer.")
       (slime-save-marker slime-output-start
         (slime-propertize-region '(face slime-repl-output-face
                                         slime-repl-output t
-                                        rear-nonsticky (face))
+                                        read-only t
```

Comment:
Maybe `read-only` could be set to the value of some new variable, e.g. `slime-repl-output-read-only`, which the user can configure? That way the default behavior would be the same (for now) but you could get read-only by tweaking your .emacs.